src/main/java/edu/harvard/dbmi/avillach/dictionary/concept/ConceptResultSetUtil.java

```diff
@@ -3,42 +3,56 @@
 import edu.harvard.dbmi.avillach.dictionary.concept.model.CategoricalConcept;
 import edu.harvard.dbmi.avillach.dictionary.concept.model.ContinuousConcept;
 import edu.harvard.dbmi.avillach.dictionary.util.JsonBlobParser;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.List;
+import java.util.Map;
```

Comment:
Instead of doubling the functions in here, why if you checked for the existence of `metadata` in the ResultSet and added it to the concept if present?

Reply:
Eh, I guess it doesn't really matter either way. nvm

Reply:
I just realized what you actually mean't with your statement. I primarily did it so I didn't have to duplicate the work done by the other method. I can alter it if you want, but then I feel like I am introducing duplication.